app/controllers/conference_registrations_controller.rb

```diff
@@ -33,8 +33,10 @@ def new
   end
 
   def show
-    @total_price = Ticket.total_price(@conference, current_user)
+    TicketPurchase.destroy_all(user_id: current_user.id, conference_id: @conference.id, paid: 'f')
```

Comment:
Actually as we want to keep just the online payment option I think that only those `ticket_purchases` should be present who have been paid, all other should be deleted. I have placed it in this controller so as immediately after `ticket_purchase` then `purchase`, you see the `conference_registration` and if somehow the payment failed, we should delete the `ticket_purchase`